engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -0,0 +1,310 @@
+            case "Sort_Direction.enso":
+                return false;
+            case "Span.enso":
+                return false;
+            case "Statistics.enso":
+                return false;
+            case "Status_Code.enso":
+                return false;
+            case "System.enso":
+                return false;
+            case "Text_Matcher.enso":
+                return false;
+            case "Text_Ordering.enso":
+                return false;
+            case "Text_Sub_Range.enso":
+                return false;
+            case "URI.enso":
+                return false;
+            case "Vector_Lexicographic_Order.enso":
+                return false;
+            case "Version.enso":
+                return false;
+            case "Warning.enso":
+                return false;
+            case "Common.enso":
+                return false;
+            case "Date.enso":
+                return false;
+            case "Date_Period.enso":
+                return false;
+            case "Date_Time.enso":
+                return false;
+            case "Day_Of_Week.enso":
+                return false;
+            case "Day_Of_Week_From.enso":
+                return false;
+            case "Duration.enso":
+                return false;
+            case "Environment.enso":
+                return false;
+            case "Existing_File_Behavior.enso":
+                return false;
+            case "Exit_Code.enso":
+                return false;
+            case "File.enso":
+                return false;
+            case "File_Format.enso":
+                return false;
+            case "File_Permissions.enso":
+                return false;
+            case "Function.enso":
+                return false;
+            case "List.enso":
+                return false;
+            case "Maybe.enso":
+                return false;
+            case "Meta.enso":
+                return false;
+            case "Pair.enso":
+                return false;
+            case "Platform.enso":
+                return false;
+            case "Problem_Behavior.enso":
+                return false;
+            case "Process.enso":
+                return false;
+            case "Random.enso":
+                return false;
+            case "Time_Of_Day.enso":
+                return false;
+            case "Time_Zone.enso":
+                return false;
+            case "Numbers.enso":
+                return false;
+            case "Vector.enso":
+                return false;
+            case "Any.enso":
+                return false;
+            case "Filter_Condition.enso":
+                return false;
+            case "Json.enso":
+                return false;
+            case "Bound.enso":
+                return false;
+            case "Rank_Method.enso":
+                return false;
+            case "Regression.enso":
+                return false;
+        }
```

Comment:
This looks a bit scary.

Can't we do `private static final List<String> blacklist = List.of("Regression.enso", ...)` and then just `blacklist.contains`?

Reply:
Done in [5fde226](https://github.com/enso-org/enso/pull/3814/commits/5fde22602199b6966e88781a043f608e531556d7)